packages/financial-templates-lib/test/clients/FinancialContractClient.js

```diff
@@ -71,28 +76,40 @@ const versionedIt = function (supportedVersions, shouldBeItOnly = false) {
 
 const Convert = (decimals) => (number) => parseFixed(number.toString(), decimals).toString();
 
-contract("FinancialContractClient.js", function (accounts) {
-  const sponsor1 = accounts[0];
-  const sponsor2 = accounts[1];
+describe("FinancialContractClient.js", function () {
+  before(async function () {
+    accounts = await web3.eth.getAccounts();
+    [sponsor1, sponsor2] = accounts;
+  });
+
+  let sponsor1, sponsor2;
 
   TESTED_CONTRACT_VERSIONS.forEach(function (contractVersion) {
     // Store the contractVersion.contractVersion, type and version being tested
     iterationTestVersion = contractVersion;
 
+    const { getAbi, getBytecode } = require(getContractsNodePackageAliasForVerion(contractVersion.contractVersion));
+
+    const createContract = (name) => {
```

Comment:
would it be useful to refactor this to a common helper? it looks like you do it in two places but I know for sure we do it in other places similar to this.

You'd likely need to pass in the `getContract` method, though, which is not ideal.

Reply:
Yeah, it's also tricky because the require needs to be called from this file since requires resolve packages based on the file where they're declared. So if the code were in common, it would try to find that package under uma/common. That might happen to work in our monorepo, but if there were name conflicts or the package weren't hoisted, it could cause issues (I think).

Reply:
ye that makes sense 100p and is a good enough reason to leave it as it is now.